<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportToDbfDialog</class>
 <widget class="QDialog" name="ExportToDbfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExportToDbfDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="CProgressBar" name="pbFiles">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
     <property name="format">
      <string>Обработано шаблонов %v/%m</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabParameters">
      <attribute name="title">
       <string>Параметры</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="grpOutFileName">
         <property name="title">
          <string>Путь выгрузки:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="edtOutPath">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>&lt;Не указан&gt;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBrowse">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>27</width>
              <height>27</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>27</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkMakeArchive">
            <property name="text">
             <string>создать архив</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="gbExportTemplates">
         <property name="title">
          <string>Шаблоны выгрузки</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbTemplateIndividual">
            <property name="text">
             <string>Индивидуальные отчеты</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cmbCodec">
            <item>
             <property name="text">
              <string>utf-8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>cp1251</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>cp866</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblCodec">
            <property name="text">
             <string>Кодировка файлов с шаблонами:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QComboBox" name="cmbExportTemplates">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>&lt;Не задано&gt;</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="rbTemplateGroups">
            <property name="text">
             <string>Группы отчетов</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Параметры шаблона</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QScrollArea" name="saTemplateParams">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>431</width>
               <height>181</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="paramNamesLayout">
                <property name="rightMargin">
                 <number>6</number>
                </property>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QVBoxLayout" name="paramValuesLayout">
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLog">
      <attribute name="title">
       <string>Лог</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textLogs"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="btnExport">
     <property name="text">
      <string>Экспорт</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="CProgressBar" name="pbRecords">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="format">
      <string>Обработано записей %v/%m</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ExportToDbfDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
